isomorphe/doc/index.md

```diff
@@ -1,13 +1,45 @@
-# ecospheres-isomorphe
-
 ## Pourquoi ISOMorphe ?
 
-Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
+Le moissonnage d'un catalogue INSPIRE par data.gouv.fr nécessite, lors du moissonnage, de convertir les fiches du catalogue d'origine ISO-19139/INSPIRE en un format compatible avec le modèle de données de data.gouv.fr (inspiré de DCAT). Pour cela, le moissonneur `csw-iso-19139` de data.gouv.fr s'appuie sur l'[algorithme de conversion ISO-19139 vers GeoDCAT-AP](https://github.com/SEMICeu/iso-19139-to-dcat-ap/tree/geodcat-ap-2.0.0) élaboré par le groupe de travail européen SEMICeu.
```

Comment:
```suggestion
Le moissonnage d'un catalogue INSPIRE par data.gouv.fr nécessite, lors du moissonnage, de convertir les fiches du catalogue d'origine ISO-19139/INSPIRE en un format compatible avec le modèle de données de data.gouv.fr. Pour cela, le moissonneur `csw-iso-19139` de data.gouv.fr s'appuie sur l'[algorithme de conversion ISO-19139 vers GeoDCAT-AP](https://github.com/SEMICeu/iso-19139-to-dcat-ap/tree/geodcat-ap-2.0.0) élaboré par le groupe de travail européen SEMICeu.
```

Nitpick

Reply:
But then why would we use ISO-19139 to GeoDCAT-AP?

Reply:
Because it's the most interoperable and future proof way to exchange metadata. It doesn't make data.gouv.fr's model inspired by DCAT. It's more like it's evolving toward DCAT. Anyway, it kind of shocked me, but I don't think anyone will notice. Nitpick.

Reply:
Applied your suggestion and tried to explain the process a bit more. WDYT?